id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py::test_case_0,functional_correctness_test_folder.csn.Qwen.no_docstr.test_snippet_26,test_case_0,functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py,,,failed,Failed: DID NOT RAISE <class 'subprocess.CalledProcessError'>,0.011567083012778312
functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py::test_case_1,functional_correctness_test_folder.csn.Qwen.no_docstr.test_snippet_26,test_case_1,functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py,,xfail,xfailed,,0.0009912920068018138
functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py::test_case_2,functional_correctness_test_folder.csn.Qwen.no_docstr.test_snippet_26,test_case_2,functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py,,xfail,xfailed,,0.0002119590062648058
functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py::test_case_3,functional_correctness_test_folder.csn.Qwen.no_docstr.test_snippet_26,test_case_3,functional_correctness_test_folder/csn/Qwen/no_docstr/test_snippet_26.py,,,failed,"AssertionError: assert ('', '', 0) == ''",0.011766833020374179
